### DISCUSSION TITLE
[NEW RELEASE] Introducing MARS5, open-source, insanely prosodic text-to-speech (TTS) model.

### DISCUSSION THREAD

**Discussion by akshhack** [yl4579/StyleTTS2#251] [Show and tell]
![mars-banner](https://github.com/yl4579/StyleTTS2/assets/21692676/84da0bdd-d78b-4157-9081-29c251f3e5cc)

Hey community members! 👋 

We, at [CAMB.AI](https://www.camb.ai), are super stoked to announce the open source release of **MARS5**, a new speech emulation model that is able to replicate even extremely tough prosody like sports commentary, anime, movies with just a few seconds of audio reference. 

_Check out our release:_ https://github.com/Camb-ai/MARS5-TTS

Watch our demo here:

https://github.com/yl4579/StyleTTS2/assets/21692676/94063f5c-3b36-4fe4-ba11-580a2bff4da6

and the full release video: https://www.youtube.com/watch?v=bmJSLPYrKtE

We're excited to hear feedback and see the community build on top of it!

Quick links:
**Discord:** https://discord.gg/ZzsKTAKM
**Github:** https://www.github.com/camb-ai/mars5-tts
**Website:** https://www.camb.ai/
**Youtube:** https://www.youtube.com/@camb-ai

**Comment by 78Alpha** [comment]
Certainly takes some guts to shoutout a personal tool on someone else's repo. However, This MARS seems to be built on similar methods to Tortoise and XTTS. So it is old, and does not use phonetic text for proper pronunciations. Additionally, it lacks a means to train or finetune your own model with the source available (Readme lacks the steps entirely). This puts it in the same realm as Bark (suno-ai) and ElevelLabs. Not end user friendly unless you use the paid service.

The only relation to StyleTTS2 is that it is a TTS model.

**Comment by jikkuatwork** [comment]
Why are you not here?

https://huggingface.co/spaces/TTS-AGI/TTS-Arena
